client_test.go

```diff
@@ -680,16 +691,42 @@ func TestRooms(t *testing.T) {
 				So(err, ShouldBeNil)
 				So(r.ID, ShouldEqual, room.ID)
 				So(r.Name, ShouldEqual, roomName)
+				So(r.PushNotificationTitleOverride, ShouldResemble, &roomPNTitleOverride)
 				So(r.Private, ShouldEqual, true)
 				So(r.MemberUserIDs, shouldResembleUpToReordering, []string{aliceID, bobID})
 				So(r.CustomData, ShouldResemble, map[string]interface{}{"foo": "bar"})
 			})
 
-			Convey("and update it", func() {
+			Convey("and update it to something else", func() {
```

Comment:
Do these test names make sense? The parent test is talking about room IDs

Reply:
I think they still do, sort of.

> we can create a room without providing an ID
>> and get it
>> and update it to something else

Maybe the parent should be more:
> we can create a room (without providing an ID)